runtime/fundamentals/workspaces.md

```diff
@@ -383,3 +383,137 @@ root and its members:
 | exports            | ❌        | ✅      |                                                                                                                                                                                                                 |
 | workspace          | ✅        | ❌      | Nested workspaces are not supported.                                                                                                                                                                            |
+
+## Running Commands Across Workspaces
+
+Deno provides several ways to run commands across all or specific workspace
+members:
+
+### Running Tests
+
+To run tests across all workspace members, simply execute `deno test` from the
+workspace root:
+
+```sh
+deno test
+```
+
+This will run tests in all workspace members according to their individual test
+configurations.
+
+To run tests for a specific workspace member, you can either:
+
+1. Change to that member's directory and run the test command:
+
+```sh
+cd my-directory
+deno test
+```
+
+2. Or specify the path from the workspace root:
+
+```sh
+deno test my-directory/
+```
+
+### Formatting and Linting
+
+Similar to testing, formatting and linting commands run across all workspace
+members by default:
+
+```sh
+deno fmt
+deno lint
+```
+
+Each workspace member follows its own formatting and linting rules as defined in
+its `deno.json` file, with some settings inherited from the root configuration
+as shown in the table above.
+
+### Using Workspace Tasks
+
+You can define tasks at both the workspace root and in individual workspace
+members:
+
+```json title="deno.json"
+{
+  "workspace": ["./add", "./subtract"],
+  "tasks": {
+    "build": "echo 'Building all packages'",
+    "test:all": "deno test"
+  }
+}
+```
+
+```json title="add/deno.json"
+{
+  "name": "@scope/add",
+  "version": "0.1.0",
+  "exports": "./mod.ts",
+  "tasks": {
+    "build": "echo 'Building add package'",
+    "test": "deno test"
+  }
+}
+```
+
+To run a task defined in a specific package:
+
+```sh
+deno task --cwd=add build
+```
+
+## Sharing and Managing Dependencies
+
+Workspaces provide powerful ways to share and manage dependencies across
+projects:
+
+### Sharing Development Dependencies
+
+Common development dependencies like testing libraries can be defined at the
+workspace root:
+
+```json title="deno.json"
+{
+  "workspace": ["./add", "./subtract"],
+  "imports": {
+    "@std/testing/": "jsr:@std/testing@^0.218.0/",
+    "chai": "npm:chai@^4.3.7"
+  }
+}
+```
+
+This makes these dependencies available to all workspace members without needing
+to redefine them.
+
+### Managing Version Conflicts
+
+The workspace root's import map takes precedence over member-specific imports,
+which helps ensure consistent versions across your workspace. For example, if
+both the root and a member specify different versions of the same dependency,
+the root's version will be used.
```

Comment:
It's the opposite. It will use the member's version when resolving the dependency within the member's folder. The member allows overwriting it. That said, when resolving dependency's outside the member's folder, it will resolve based on the workspace root's folder.

Member deno.json dependencies are restricted to the specific member's folder, but the workspace root's deno.json applies to the entire application (including jsr and https dependencies)

Reply:
Have updated to:

When resolving dependencies, workspace members can override dependencies defined
in the root. If both the root and a member specify different versions of the
same dependency, the member's version will be used when resolving within that
member's folder. This allows individual packages to use specific dependency
versions when needed.

However, member-specific dependencies are scoped only to that member's folder.
Outside of member folders, or when working with files at the workspace root
level, the workspace root's import map will be used for resolving dependencies
(including JSR and HTTPS dependencies).